modules/aws-node-termination-handler/main.tf

```diff
@@ -0,0 +1,26 @@
+locals {
+  release_name = "aws-node-termination-handler"
+  chart_name   = "aws-node-termination-handler"
+  namespace    = var.namespace
+  repository   = "https://aws.github.io/eks-charts"
+}
+
+
+resource "helm_release" "node_termination_handler" {
+  count            = var.enable ? 1 : 0
+  name             = local.release_name
+  chart            = local.chart_name
+  version          = var.chart_version
+  repository       = local.repository
+  namespace        = local.namespace
+  create_namespace = true
```

Comment:
nitpick: why keep _some_ of the values as `locals` while `create_namespace` and `wait` are set directly on the `helm_release`?